test/FundraisingBanner.test.js

```diff
@@ -48,7 +48,7 @@ describe('FundraisingBanner Component', () => {
     });
 
     it('should set the background-image url property to match the config file sources', () => {
-      const styles = component.find('.fundraisingBanner').prop('style');
+      const styles = component.find('div.fundraisingBanner').prop('style');
```

Comment:
The components themselves seem to be rendered as the wrapper even though it is mounted, i.e. 
```
<FundraisingBanner className="fundraisingBanner" ... >
  <div className="fundraisingBanner" ...>
...
```
This can be updated later.